macOS/LocalPackages/HistoryView/Sources/HistoryView/DataModel.swift

```diff
@@ -93,13 +93,19 @@ public enum DataModel {
         }
     }
 
+    public enum HistoryQuerySource: String, Codable {
+        case initial, user, auto
```

Comment:
This was added to support firing pixels only when the user triggers a history fetch.